AutoMLOps/AutoMLOps.py

```diff
@@ -173,14 +176,27 @@ def generate(project_id: str,
     _create_dockerfile()
     if not run_local:
         CloudRunBuilder.formalize(TOP_LVL_NAME, DEFAULTS_FILE)
+    if use_terraform:
+        TerraformBuilder.formalize(TOP_LVL_NAME, DEFAULTS_FILE, run_local)
+    else:
+        _create_resources_scripts(run_local)
 
-def run(run_local: bool):
+def run(run_local: bool = True,
+        use_terraform: bool = False):
     """Builds, compiles, and submits the PipelineJob.
 
     Args:
         run_local: Flag that determines whether to use Cloud Run CI/CD.
+        use_terraform: Flag that determines whether to create resources using Terraform or a shell script.
     """
-    BuilderUtils.execute_process('./'+RESOURCES_SH_FILE, to_null=False)
+    if use_terraform:
+        os.chdir(TOP_LVL_NAME + 'terraform/state_bucket/')
+        BuilderUtils.execute_process('./'+ 'state_bucket_runner.sh', to_null=False)
+        os.chdir('../environment/')
+        BuilderUtils.execute_process('./'+ 'terraform_runner.sh', to_null=False)
+        os.chdir('../../')
```

Comment:
Per this issue - need to cd out one more directory

![image](https://user-images.githubusercontent.com/104024446/224823995-e6110af7-6f74-4f93-99cb-24bd515a13e8.png)